From 12a91d0e465bffa129f67b4b04162743c4843027 Mon Sep 17 00:00:00 2001
From: Zhen Lei <thunder.leizhen@huawei.com>
Date: Wed, 27 Apr 2022 18:48:33 +0800
Subject: [PATCH] arm64: kdump: Update the name of crashk_low_res
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 12a91d0e465bffa129f67b4b04162743c4843027
Modified-by-SEL: No


hulk inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I545H8
CVE: NA

-------------------------------------------------------------------------

To be consistent with the style of other ARCHs such as x86, the kexec
commit b5a34a20984c ("arm64: support more than one crash kernel regions")
requires all crash regions to be named "Crash kernel". Update the name of
crashk_low_res, so that we can directly use the latest kexec tool without
having to maintain a private version.

Signed-off-by: Zhen Lei <thunder.leizhen@huawei.com>
Reviewed-by: Kefeng Wang <wangkefeng.wang@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/kernel/setup.c | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/arch/arm64/kernel/setup.c b/arch/arm64/kernel/setup.c
index dcbefb4a9736..f6e56847a4e9 100644
--- a/arch/arm64/kernel/setup.c
+++ b/arch/arm64/kernel/setup.c
@@ -273,16 +273,9 @@ static void __init request_standard_resources(void)
 			request_memmap_resources(res);
 
 #ifdef CONFIG_KEXEC_CORE
-		/*
-		 * Userspace will find "Crash kernel" or "Crash kernel (low)"
-		 * region in /proc/iomem.
-		 * In order to distinct from the high region and make no effect
-		 * to the use of existing kexec-tools, rename the low region as
-		 * "Crash kernel (low)".
-		 */
+		/* Userspace will find "Crash kernel" region in /proc/iomem. */
 		if (crashk_low_res.end && crashk_low_res.start >= res->start &&
 				crashk_low_res.end <= res->end) {
-			crashk_low_res.name = "Crash kernel (low)";
 			request_resource(res, &crashk_low_res);
 		}
 		if (crashk_res.end && crashk_res.start >= res->start &&
-- 
2.34.1

